Userland/Libraries/LibJS/Heap/Heap.h

```diff
@@ -101,8 +101,9 @@ class Heap : public HeapBase {
         }
     }
 
-    size_t m_max_allocations_between_gc { 100000 };
-    size_t m_allocations_since_last_gc { 0 };
+    static constexpr size_t m_gc_min_bytes_threshold { 4 * 1024 * 1024 };
```

Comment:
Rather than the `m_` prefix for this constant, we use `SCREAMING_CASE` (see https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names).

Reply:
That's true, I guess I got confused because of something in BlockAllocator.